From a4b61995bbe3d4c4ea1483ecb9a189957ac901e9 Mon Sep 17 00:00:00 2001
From: Zoran Stojsavljevic <zoran.stojsavljevic@gmail.com>
Date: Sat, 19 Jun 2021 17:53:21 +0200
Subject: [PATCH] ARM: DT: Enable symbols when CONFIG_OF_OVERLAY is used

---
 arch/arm/boot/Makefile     | 4 ++++
 1 files changed, 4 insertions(+)

diff --git a/arch/arm/boot/Makefile b/arch/arm/boot/Makefile
index 0b3cd7a33..87a8b8db3 100644
--- a/arch/arm/boot/Makefile
+++ b/arch/arm/boot/Makefile
@@ -29,6 +29,10 @@ export ZRELADDR INITRD_PHYS PARAMS_PHYS
 
 targets := Image zImage xipImage bootpImage uImage
 
+ifeq ($(CONFIG_OF_OVERLAY),y)
+DTC_FLAGS += -@
+endif
+
 ifeq ($(CONFIG_XIP_KERNEL),y)
 
 cmd_deflate_xip_data = $(CONFIG_SHELL) -c \
-- 
2.31.1

