From e4087fe60a1e59b13b4967e5958f6f18fd32de9d Mon Sep 17 00:00:00 2001
From: Zoran Stojsavljevic <zoran.stojsavljevic@gmail.com>
Date: Fri, 9 Jul 2021 15:59:04 +0200
Subject: [PATCH] hack-gpiolib-yes-we-have-drivers-stomping-on-each-ot.patch

---
 drivers/gpio/gpiolib.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpio/gpiolib.c b/drivers/gpio/gpiolib.c
index 6367646dc..d3215067a 100644
--- a/drivers/gpio/gpiolib.c
+++ b/drivers/gpio/gpiolib.c
@@ -1847,9 +1847,9 @@ static int gpiod_request_commit(struct gpio_desc *desc, const char *label)
 
 	if (test_and_set_bit(FLAG_REQUESTED, &desc->flags) == 0) {
 		desc_set_label(desc, label ? : "?");
-	} else {
-		ret = -EBUSY;
-		goto out_free_unlock;
+//	} else {
+//		ret = -EBUSY;
+//		goto out_free_unlock;
 	}
 
 	if (gc->request) {
-- 
2.31.1

